### PR TITLE
chore(); schema; update schemas with directives

### PR DESCRIPTION
**Pull request by dhruv-chauhan**

### CODE REVIEW

**Comment by dmelotik** [comment]
@dhruv-chauhan could you focus on perpfutures in this PR? No need to scrap any work, but do those first and we will merge that in. Thanks! Also, no need to update the versions

cc @lowjiansheng

File: schema-bridge.graphql [hunk 1]
```diff
@@ -982,7 +982,7 @@ type PoolHourlySnapshot @entity(immutable: true) {
   rewardTokenEmissionsUSD: [BigDecimal!]
 }
 
-type PoolRouteSnapshot @entity(immutable: true) {
+type PoolRouteSnapshot @entity(immutable: true) @regularPolling {
```
Comment (dmelotik) [inline schema-bridge.graphql:985]: @lowjiansheng what do you think here? This is both a daily and hourly snapshot...

Comment (lowjiansheng) [reply schema-bridge.graphql:985]: I think having it as an hourlySnapshot would be good. We'll just poll the data every hour instead of every day.

**Review by dmelotik — APPROVED** [review]
lgtm, let's merge, this. Then finish the rest in a different PR. Thanks @dhruv-chauhan

**Pull request merged**